tests/boms.rs

```diff
@@ -122,7 +122,7 @@ async fn update_bom_with_invalid_input_returns_bad_request() {
         .await;
 
     let added_bom = app
-        .post_bom(&vec![comp])
+        .post_bom(&[comp])
```

Comment:
we can use a slice directly here -> https://rust-lang.github.io/rust-clippy/master/index.html#useless_vec